apps/builder/app/canvas/collapsed.ts

```diff
@@ -0,0 +1,122 @@
+import { collapsedAttribute } from "@webstudio-is/react-sdk";
```

Comment:
could probably go to ./shared/collapse-detection.ts

Reply:
Do we need this to be used elsewhere?

Reply:
nope, in that case features/collapse-detection